lib/rails_translation_manager.rb

```diff
@@ -10,6 +10,7 @@
 require 'rails_translation_manager/locale_checker/missing_english_locales'
 require 'rails_translation_manager/locale_checker/plural_forms'
 require 'rails_translation_manager/locale_checker/incompatible_plurals'
+require 'rails_translation_manager/locale_checker/all_locales'
```

Comment:
In the commit message, shouldn't this be:

```diff
- config/locales/*/*.yml
+ config/locales/**/*.yml
```

Reply:
I don't think so as `config/locales/*/*.yml` works fine for the path expansion, maybe `config/locales/**/*.yml` works too...

Reply:
Ah, I always use `**` out of habit. I suspect `*` just handles one directory down, whereas `**` handles infinitely nested directories. Anyhow, no biggie.